### ISSUE TITLE
Accessing list elements

### ISSUE THREAD

**Issue by ghost** [defunkt/pystache#134]
I'm trying to use lists as input data:

```
{'l': ['v1', 'v2']}
```

using template:

```
{{#l}}
v1:{{0}} v2:{{1}}
{{/l}}
```

and it seems doesn't work. 
Can I use lists as input data without transforming it to dict?

**Comment by cjerdonek** [comment]
No, you currently can't.  The tag key is always interpreted as the key to a hash (dict) or method name.  Maybe this could be added though in a sensible way.

**Comment by cjerdonek** [comment]
See also issue #143.
